lib/rubocop/cop/variable_force/assignment.rb

```diff
@@ -109,8 +109,14 @@ def operator_assignment_node
           node.parent
         end
 
+        # rubocop:disable Metrics/CyclomaticComplexity
         def multiple_assignment_node
-          return nil unless node.parent&.mlhs_type?
+          return nil unless (candidate_mlhs_node = node.parent)
+
+          # In `(foo, bar), *baz`, the splat node must be traversed as well.
+          candidate_mlhs_node = candidate_mlhs_node.parent if candidate_mlhs_node.splat_type?
+
+          return nil unless candidate_mlhs_node.mlhs_type?
           return nil unless (grandparent_node = node.parent&.parent)
```

Comment:
Removing this safe navigation will allow the `rubocop:disable` annotation to be removed :-)

```suggestion
          return nil unless (grandparent_node = node.parent.parent)
```

Reply:
Oops, I missed this one. Indeed they are both gone with that, I believe this is safe to do.